src/components/calcite-block/calcite-block.stories.ts

```diff
@@ -0,0 +1,118 @@
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+import { Attributes, createComponentHTML as create, darkBackground, parseReadme } from "../../../.storybook/utils";
+import blockReadme from "./readme.md";
+import sectionReadme from "../calcite-block-section/readme.md";
+
+export default {
+  title: "calcite-block",
+  decorators: [withKnobs],
+  parameters: {
+    notes: {
+      block: parseReadme(blockReadme),
+      section: parseReadme(sectionReadme)
+    },
+    backgrounds: darkBackground
+  }
+};
+
+const createBlockAttributes: () => Attributes = () => {
+  const group = "block";
+  const dirOptions = ["ltr", "rtl"];
+  const themeOptions = ["light", "dark"];
+
+  return [
+    {
```

Comment:
question: In the future, it seems like we maybe could auto generate these from docs JSON generated by Stencil. I remember Paul maybe tried that in `calcite-components` and thought it wasn't worth it because its hard to filter some stuff out. Maybe we could have a filter map file to hide the props we don't want to see. Thoughts?

Reply:
I think we could automate a base set of options, but stories will still need to be set up manually. We can do this in a follow-up issue.

Reply:
Sounds good